Detections/AuditLogs/AdminPromoAfterRoleMgmtAppPermissionGrant.yaml

```diff
@@ -0,0 +1,74 @@
+id: f80d951a-eddc-4171-b9d0-d616bb83efdc
+name: Admin promotion after Role Management Application Permission Grant
+description: |
+  'This rule looks for a service principal being granted the Microsoft Graph RoleManagement.ReadWrite.Directory (application) permission before being used to add an Azure AD object or user account to an Admin directory role (i.e. Global Administrators). 
```

Comment:
Can you add in why this is something to alert on?

Reply:
This is the current description now:

This rule looks for a service principal being granted the Microsoft Graph RoleManagement.ReadWrite.Directory (application) permission before being used to add an Azure AD object or user account to an Admin directory role (i.e. Global Administrators).
  This is a known attack path that is usually abused when a service principal already has the AppRoleAssignment.ReadWrite.All permission granted. This permission Allows an app to manage permission grants for application permissions to any API.
  A service principal can promote itself or other service principals to admin roles (i.e. Global Administrators). This would be considered a privilege escalation technique.